docs/index.rst

```diff
@@ -58,3 +58,5 @@ In a terminal, run the following:
    * :ref:`genindex`
    * :ref:`modindex`
    * :ref:`search`
+
+.. image:: ./images/installation.gif
```

Comment:
Can you specify `alternative text` and size options as well? See https://sublime-and-sphinx-guide.readthedocs.io/en/latest/images.html

Reply:
Done.